package.json

```diff
@@ -14,5 +14,9 @@
 		"envs": [
 			"browser"
 		]
+	},
+	"dependencies": {
```

Comment:
I’m not sure but I think this should be `bundledDependencies`

Reply:
They are bundled, so they are not technically dependencies at all, in the npm sense. So they should be removed from package.json.

Reply:
Actually, instead of bundling the code here, we can just put a package.json in the `extensions` folder, add them as `bundledDependencies`. That way they can be easily updated using npm, but are also bundled, like now.